modules/quality_control/readme.md

```diff
@@ -0,0 +1,28 @@
+## Imaging Quality Control Module
+
+#### Purpose
+The purpose of this module is to verify and help ensure imaging data has been uploaded into LORIS and reviwed for Quality Control (QC). It provides a queryable interface for users to gauge the completeness of imaging data collections matched against a given protocol, and the status of QC review processes on them.
+
+Note: this module is not the module in which QC is performed on images, just the module in which QC is *reviewed*.  QC is actually marked on the images in the Imaging Browser module.
+
+#### Intended Users
+This module is used by study administrators, imaging scientists, analysts, coordinators, and clinicians.
+
+#### Scope
+The Imaging QC module allows you to view specific data on uploaded image files, with many additional filtering options. The resulting table has a column for QC status, so you can see whether the upload has passed QC or not. The table provides a link to each file. You can download the table in `.csv` format. 
+
+Note: To QC a scan, the user must click the link leading to the Imaging Browser, and enter the QC details in that module.
+
+#### Permissions
+`Placeholder - waiting for contribution from LL`
+
+#### Interactions
+The Imaging Quality Control module is directly related to the following modules:
+
+##### Imaging Browser
+The Imaging QC module displays images that have been loaded in LORIS and QC'd in the Imaging Browser. The Imaging Browser also provides metadata and visualization of each scan and session collected for a study. 
+
+#### Imaging Uploader
+This module displays the status of images that have been excluded from or failed to load into LORIS, for a variety of reasons.  
+#### Configuration
+`Placeholder - waiting for contribution from LL`
```

Comment:
Needs updating before merge.

Reply:
`tblScanTypes` is the only config i could find. It allows the module to display a chosen subset of scan types on the front end.